apis/configwatch.go

```diff
@@ -33,9 +36,11 @@ import (
 )
 
 type Server struct {
-	KFRTConfigs *kf.NFConfigs
-	HostName    string
-	l3afdServer *http.Server
+	KFRTConfigs   *kf.NFConfigs
+	HostName      string
+	l3afdServer   *http.Server
+	CaCertPool    *x509.CertPool
+	SANMatchRules string
```

Comment:
this should be a string array and we can split using some delimiter, such as ',' when loading from config file.